downloads/swan-lake-release-notes/swan-lake-2201.5.0/RELEASE_NOTE.md

```diff
@@ -354,6 +356,9 @@ To view bug fixes, see the [GitHub milestone for Swan Lake 2201.5.0](https://git
 
 ### New features
 
+#### Installers
+- Introduced a [new installer](https://dist.ballerina.io/downloads/2201.5.0/ballerina-2201.5.0-swan-lake-macos-arm-x64.pkg) that supports the macOS-ARM architecture. 
```

Comment:
```suggestion
- Introduced a [new installer](https://dist.ballerina.io/downloads/2201.5.0/ballerina-2201.5.0-swan-lake-macos-arm-x64.pkg) that supports the macOS-ARM architecture. 
```
Shouldn't we remove it from here as we are mentioning it under the `Install Ballerina` section?